### PR TITLE
Community moderator labels

### PR DESCRIPTION
**Pull request by robbiecarlton**
@levity - this is how it looks, using the 'subtitles' param in PersonCards

screen shot 2016-03-08 at 2 40 38 pm

### CODE REVIEW

File: src/containers/community/CommunityMembers.js [hunk 1]
```diff
@@ -23,12 +24,14 @@ const fetch = fetchWithCache(fetchPeople)
     ...connectedListProps(state, {subject, id, query}, 'people'),
     community: state.communities[id],
     currentUser: state.people.current,
-    error: findError(state.errors, FETCH_PEOPLE, 'peopleByQuery', cacheId)
+    error: findError(state.errors, FETCH_PEOPLE, 'peopleByQuery', cacheId),
+    moderators: state.peopleByQuery[qs.stringify({subject: 'community-moderators', id})] || []
```
Comment (levity) [inline src/containers/community/CommunityMembers.js]: can you make this moderatorIds for clarity?

Comment (levity) [inline src/containers/community/CommunityMembers.js]: can you rename this to `moderatorIds` for clarity?

File: src/containers/community/CommunityMembers.js [hunk 2]
```diff
@@ -53,10 +56,11 @@ export default class CommunityMembers extends React.Component {
   }
 
   render () {
-    let { pending, people, location: { query }, community, currentUser, error } = this.props
+    let { pending, people, moderators, location: { query }, community, currentUser, error } = this.props
     if (error) return <AccessErrorMessage error={error}/>
     if (!currentUser) return <div>Loading...</div>
     let { search } = query
+    let subtitles = moderators.reduce((acc, mod) => ({...acc, [mod]: 'Moderator'}), {})
```
Comment (levity) [inline src/containers/community/CommunityMembers.js]: i think it'll be more flexible in terms of styling if we just stick `isModerator` on each of the corresponding objects in `person` and then check for it in `PersonCard`. my first thought when i saw the screenshot was "that should be lighter gray (like .meta)" and the second thought was "we should show bios there"

**Comment by robbiecarlton** [comment]
screen shot 2016-03-08 at 4 41 47 pm
@levity - like this?

**Comment by robbiecarlton** [comment]
@levity - good to merge?

**Comment by levity** [comment]
:+1:

**Pull request merged**